ledger/src/shred/shred_code.rs

```diff
@@ -136,7 +134,7 @@ pub(super) fn sanitize<T: ShredCodeTrait>(shred: &T) -> Result<(), Error> {
     }
     let common_header = shred.common_header();
     let coding_header = shred.coding_header();
-    if common_header.index as usize >= MAX_CODE_SHREDS_PER_SLOT {
+    if common_header.index as usize >= MAX_DATA_SHREDS_PER_SLOT.max(MAX_CODE_SHREDS_PER_SLOT) {
```

Comment:
Thanks for moving the constant to `shred.rs`; one more question, what's thinking here with doing `.max()` ?

Reply:
Well assert here is checking if index could be valid, but same index is used for both code and data shreds so this looked appropriate.

Reply:
Gotcha, I think this comes back to @bw-solana's comment in the other spot:
> ... Using MAX_DATA_SHREDS_PER_SLOT to bound coding shreds is confusing

Since we decided to keep distinct constants, I think it makes sense to only `MAX_CODE` here

Reply:
Yeah true it just hit me this is in a file called shred_code.rs ... Thanks for guiding me back to the light!